docs/live/apply.md

```diff
@@ -40,4 +30,89 @@ Flags:
     
   wait-timeout:
     The threshold for how long to wait for all resources to reconcile before
-    giving up. The default value is 1 minute.
\ No newline at end of file
+    giving up. The default value is 1 minute.
+
+#### `kubectl apply` vs `kpt live apply`
+
+|                     | `kubectl apply`            | `kpt live apply`          |
+|---------------------|----------------------------|---------------------------|
+|Usage                | kubectl apply -f /dir      | kpt live apply /dir       |
+|Applied resource set | Not tracked                | Tracked                   |
+|Prune                | Imperative and error prone | Declarative and reliable  |
+|wait-for-reconcile   | Not supported              | Supported                 |
+
+Usage:
+  In terms of usage, both `kubectl apply` and `kpt live apply` follow similar pattern.
+  The user experience remains unchanged.
+
+Applied resource set(ARS):
+  `kpt live apply` tracks the state of your applied resource set and related configuration. This
+  helps `kpt` to reliably reconcile the real world ARS with your configuration changes.
+
+Prune:
```

Comment:
I think this section might still fit better in the description of `live` rather than here. The concepts do affect almost all the `live` commands, although I agree they are most important for `apply`

Reply:
Yeah, apply is the main use case and the example is also focusing on apply. The rest should be intuitive and we can always keep adding stuff.